composer.json

```diff
@@ -1,65 +1,74 @@
 {
-    "name": "zfcampus/zf-hal",
-    "description": "ZF2 Module providing Hypermedia Application Language assets and rendering",
-    "type": "library",
-    "license": "BSD-3-Clause",
-    "keywords": [
-        "zf2",
-        "zend",
-        "module",
-        "rest",
-        "hal"
-    ],
-    "homepage": "http://apigility.org/",
-    "support": {
-        "email": "apigility-users@zend.com",
-        "irc": "irc://irc.freenode.net/apigility",
-        "source": "https://github.com/zfcampus/zf-hal",
-        "issues": "https://github.com/zfcampus/zf-hal/issues"
+  "name": "zfcampus/zf-hal",
+  "description": "ZF2 Module providing Hypermedia Application Language assets and rendering",
+  "type": "library",
+  "minimum-stability": "beta",
+  "license": "BSD-3-Clause",
+  "keywords": [
+    "zf2",
+    "zend",
+    "module",
+    "rest",
+    "hal"
+  ],
+  "homepage": "http://apigility.org/",
+  "support": {
+    "email": "apigility-users@zend.com",
+    "irc": "irc://irc.freenode.net/apigility",
+    "source": "https://github.com/zfcampus/zf-hal",
+    "issues": "https://github.com/zfcampus/zf-hal/issues"
+  },
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://packages.zendframework.com"
```

Comment:
In the future, please, please, please do not reformat this file. Normal spacing is 4 spaces per level of indentation. Changing it to two spaces makes understanding the changes quite difficult, as every line represents a change!